app/components/schools/participants/status_table_row.rb

```diff
@@ -26,9 +26,7 @@ def participant_is_on_a_cip?
         profile.school_cohort.school_chose_cip?
       end
 
-      def date_column_value
```

Comment:
Can this just be removed and the have an empty value in the <td> where it currently is?